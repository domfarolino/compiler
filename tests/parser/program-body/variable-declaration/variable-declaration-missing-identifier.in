program dominic is

  integer dominic;
  float;

begin end program.
