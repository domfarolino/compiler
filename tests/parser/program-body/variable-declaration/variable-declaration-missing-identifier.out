[1;31mLine 4: Expected identifier in variable declaration, but got: ';'
Line 4: Expected 'begin' in program body, but got: ';'
Line 4: Error parsing program body
[0m